test/runtests.jl

```diff
@@ -88,6 +89,13 @@ df, names = create_columns_from_iterabletable(load(filename, "Sheet1!C4:O7", hea
 @test DataValues.isna(df[12][4])
 @test df[13] == [NA, 3.4, "HKEJW", NA]
 
+# Test for saving DataFrame to XLSX
+input = (Day=["Nov. 27","Nov. 28","Nov. 29"], Highest=[78,79,75]) |> DataFrame
+file = save("file.xlsx", input)
+output = load("file.xlsx", "Sheet1!A1:B4", header=true) |> DataFrame
```

Comment:
Can we change this line to:
````julia
output = load("file.xlsx", "Sheet1") |> DataFrame
````
It should automatically detect that there are headers, and also auto-detect the range with the data.